resources/group.go

```diff
@@ -225,6 +224,20 @@ func (obj *GroupRes) CheckApply(apply bool) (checkOK bool, err error) {
 type GroupUID struct {
 	BaseUID
 	name string
+	gid  *uint32
+}
+
+// IFF aka if and only if they are equivalent, return true. If not, false.
+func (obj *GroupUID) IFF(uid ResUID) bool {
+	res, ok := uid.(*GroupUID)
+	if !ok {
+		return false
+	}
+	var eq bool
+	if obj.gid != nil && res.gid != nil {
+		eq = *obj.gid == *res.gid
```

Comment:
cute, but i think this is overly complex
just return true/false where appropriate